acceptance/tests/peering/peering_connect_test.go

```diff
@@ -350,7 +368,7 @@ func TestPeering_Connect(t *testing.T) {
 				// If ACLs are enabled, test that deny intentions prevent connections.
 				if c.ACLsEnabled {
 					logger.Log(t, "testing intentions prevent connections through the terminating gateway")
-					k8s.CheckStaticServerConnectionFailing(t, staticClientOpts, staticClientName, externalServerHostnameURL)
```

Comment:
@t-eckert this was the line causing the issue with the test. It should be checking for 403